### PR TITLE
feat: update web workflow to rename it on nightly job

### PR DESCRIPTION
**Pull request by hamzabessa**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by hamzabessa** [comment]
added run-name to rename the web nightly job

File: .github/workflows/web.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: web
-
+run-name: ${{ github.event_name == 'schedule' && format('web nightly {0}', github.ref ) || github.event_name == 'pull_request' && github.event.pull_request.title || github.event_name == 'push' && github.event.head_commit.message || 'unknown' }}
```
Comment (kduret) [inline .github/workflows/web.yml:2]:
```suggestion
run-name: ${{ github.event_name == 'schedule' && format('web nightly {0}', github.ref ) || github.event_name == 'pull_request' && github.event.pull_request.title || github.event_name == 'push' && github.event.head_commit.message || github.event_name == 'workflow_dispatch' && github.event.workflow || 'unknown' }}
```

**Review by kduret — COMMENTED** [review]
you could try to separate conditions on multilines : https://github.com/orgs/community/discussions/25641#discussioncomment-3248571

File: .github/workflows/web.yml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 name: web
-
+run-name: ${{ github.event_name == 'schedule' && format('web nightly {0}', github.ref ) || 
+          github.event_name == 'pull_request' && github.event.pull_request.title || 
+          github.event_name == 'push' && github.event.head_commit.message || 
+          github.event_name == 'workflow_dispatch' && github.event.workflow || 'unknown' }}
```
Comment (kduret) [inline .github/workflows/web.yml:5]:
```suggestion
          github.event_name == 'workflow_dispatch' && github.workflow || 'unknown' }}
```

**Review by kduret — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**